test/controllers.test.js

```diff
@@ -34,12 +34,12 @@ describe('GET /callback', () => {
       });
   });
 
-  it('should return 500 when using an invalid code', (done) => {
+  it('should return 400 when using an invalid code', (done) => {
```

Comment:
I didn't know It was a 400 in this case. Can you show me the associated documentation ?

Reply:
Bad request